libkbfs/tlf_journal.go

```diff
@@ -1201,11 +1199,10 @@ func (j *tlfJournal) removeFlushedMDEntry(ctx context.Context,
 }
 
 func (j *tlfJournal) flushOneMDOp(
-	ctx context.Context, end MetadataRevision,
-	maxMDRevToFlush MetadataRevision) (flushed bool, err error) {
+	ctx context.Context, maxMDRevToFlush MetadataRevision) (
+	flushed bool, err error) {
 	if maxMDRevToFlush == MetadataRevisionUninitialized {
-		// Short-cut `getNextMDEntryToFlush`, which would otherwise read
-		// an MD from disk and sign it unnecessarily.
+		// Nothing to do.
```

Comment:
Ah, yeah, it wasn't accurate anymore. The old code was:

1) Read MD,
2) Check against `end`;
3) Sign MD;
4) Check against `maxMDRevToFlush`.

The new code is 1) and 4), so now that test just saves the read. Add a similar comment.